Lexicon.md

```diff
@@ -4,19 +4,23 @@
 * __Bit String__: A sequence of __Bits__ of arbitrary length $$ B^n := (b_k)_{k=0}^n , \  k \in \mathbb{N} $$.
 * __Vulcan Set__: The set of all __Bit Strings__ of infinite length $$ \mathbb{V} := \{ B_0^{\infty}, B_1^{\infty}, \dots, B_{\infty}^{\infty} \} , \ B_i^{\infty} \neq B_j^{\infty} $$.
 * __Information__: Anything that can be encoded into a __Bit String__.
-* __Environment__: __Information__ that is defined relative to a particular __Agent__. It is seperated from an __Agent__ by an __Agent Boundary__, and encompasses all content needed to process any relevant __Actions__ and __Observations__.
+* __Environment__: An __Object__ that is defined relative to a particular __Actor__. It is seperated from an __Actor__ by an __Actor Boundary__, and encompasses all content needed to process any relevant __Events__.
 * __World__: Synonymous with __Environment__.
```

Comment:
FWIW I think of World as the part of the Environment modeled by a particular Agent.

Reply:
Right, I was suggesting that World and World Model are synonymous. Do you differentiate between Environment and World Model?

Reply:
It's true that usually when we say 'World' in everyday speech we refer to our World Model and not the Environment. I'd be open to making that change, though it might be seen as confusing. Right now World Model is defined on line 18.

Reply:
OK to defer